modules/local/locidex_report.nf

```diff
@@ -0,0 +1,36 @@
+/* Locidex report from the seq-store
+
+*/
+
+
+process LOCIDEX_REPORT {
+    tag "$meta.id"
+    label "process_low"
+    container "${workflow.containerEngine == 'singularity' || workflow.containerEngine == 'apptainer' ? task.ext.containers.get('singularity') : task.ext.containers.get('docker')}"
+
+    input:
+    tuple val(meta), path(seq_store)
+
+    output:
+    tuple val(meta), path(output_name), emit: report
+    path "versions.yml", emit: versions
+
+    script:
+    output_name = "${meta.id}${params.locidex.report_suffix}"
+    """
+    locidex report -i $seq_store -o . --name ${meta.id} \\
```

Comment:
indeed, I had some issues with file names I aim to resolve in the next commit.

Reply:
fixed in: be8cd5bdd64c18915c8a90accba7d58d6bedc7d3

Reply:
Leaving `-o .` is okay since that is how it works now.

What did you mean by resolve though? Since it still is set to `-o .`?

Reply:
apologies, I meant resolve the old comment. The `-o  .` is too get around the issue with paths, but this can be changed when a new version of locidex is released.